static/scripts/coursesOverview.js

```diff
@@ -26,9 +26,9 @@ $(document).ready(function () {
                 courseMembers = courseMembers + '</ol>';
             }
 
-            populateModal($memberModal, '.modal-title', 'Teilnehmer des Kurses: '.concat(courseName));
+            populateModal($memberModal, '.modal-title', $t('courses.headline.courseAttendees').concat(courseName));
```

Comment:
```suggestion
            populateModal($memberModal, '.modal-title', `${$t('courses.headline.courseAttendees')} `.concat(courseName));
```
take care of the now missing space here